msal4j-sdk/src/test/java/com/microsoft/aad/msal4j/AadInstanceDiscoveryTest.java

```diff
@@ -186,4 +188,137 @@ public void aadInstanceDiscoveryTest_AutoDetectRegion_NoRegionDetected() throws
         Assert.assertTrue(entry.aliases().contains("login.microsoft.com"));
         Assert.assertTrue(entry.aliases().contains("sts.windows.net"));
     }
+
+    @DataProvider(name = "aadClouds")
+    private static Object[][] getAadClouds(){
+        return new Object[][] {{"https://login.microsoftonline.com/common"} , // #Known to Microsoft
+                {"https://private.cloud/foo"}//Private Cloud
+                };
+    }
+
+    @DataProvider(name = "b2cAdfsClouds")
+    private static Object[][] getNonAadClouds(){
+        return new Object[][] {{"https://contoso.com/adfs"}//ADFS
+//                {"https://login.b2clogin.com/contoso/b2c_policy"},//B2C
```

Comment:
Since B2C is commented out, are B2C and ADFS expected to behave the same way?

Reply:
In this Instance Discovery context, neither B2C nor ADFS provides Instance Discovery endpoint.

Reply:
That's right. Instance discovery should not be done for B2C or ADFS even if instance_discovery flag is true.